datafusion/sql/src/unparser/expr.rs

```diff
@@ -617,31 +617,55 @@ impl Unparser<'_> {
     }
 
     fn get_field_to_sql(&self, args: &[Expr]) -> Result<ast::Expr> {
-        if args.len() != 2 {
-            return internal_err!("get_field must have exactly 2 arguments");
-        }
```

Comment:
I think we don't need to remove the condition. We still need to ensure the length of arguments for `get_field` is 2.